pkgs/tools/inputmethods/ibus/dont-build-input-method-unstable-v1.patch

```diff
@@ -0,0 +1,19 @@
+diff -ur a/client/wayland/Makefile.am b/client/wayland/Makefile.am
+--- a/client/wayland/Makefile.am
++++ b/client/wayland/Makefile.am
+@@ -80,11 +80,11 @@
+ # for multiple protocols.
+ wayland_text_protocoldir = $(WAYLAND_PRTCLS_DIR)/$(WAYLAND_TEXT_PRTCL_SUBDIR)
+ 
+-input-method-unstable-v1-client-protocol.h: $(wayland_protocoldir)/input-method-unstable-v1.xml
+-	$(AM_V_GEN)$(wayland_scanner) client-header $< $@
++input-method-unstable-v1-client-protocol.h:
++	touch $@
```

Comment:
This is weird.  Looking into why the variable is not set.

The following invocation should set it:

https://github.com/ibus/ibus/blob/d0ad4e6e519439f034d08fd6531403027bd7fbde/configure.ac#L384
https://gitlab.freedesktop.org/wayland/wayland/-/blob/6137c8c21383d7ebb92cf3331be56ca0aecccd5d/wayland-scanner.m4#L12

Maybe the m4 file is not included but I am not really familiar with Autotools.

Reply:
Looks like the missing part is `wayland-scanner` in `nativeBuildInputs` (which contains the `wayland-scanner.m4`). With that, the build succeeds even without the patch.